metadata.json

```diff
@@ -14,15 +14,15 @@
   "dependencies": [
     {
       "name": "puppet/archive",
-      "version_requirement": ">= 1.0.0 < 4.0.0"
+      "version_requirement": ">= 1.0.0 < 5.0.0"
     },
     {
       "name": "puppet/staging",
       "version_requirement": ">= 2.0.1 < 4.0.0"
     },
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.25.0 < 6.0.0"
```

Comment:
Should these 2 be done separate as well for the changelog?

Reply:
I just picked a commit from @alexjfisher and do not think that two separate commits would make a difference:
* https://github.com/voxpupuli/puppet-jira/pull/320/commits/78052cd8bc317c018c41d61590a00dc02b5dbcad